src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  width: 200px;
+  box-sizing: border-box;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+    background-size: cover;
+    background-position: center;
+  }
+}
+
+.content {
+  background-position: center;
+
+  &__title {
+    display: flex;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    color: #060b35;
+  }
+
+  &__description {
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin: 0 0 16px;
+  }
+}
+
+.mark {
+  display: flex;
+  height: 16px;
+  margin-bottom: 24px;
+  justify-content: space-between;
+  align-items: center;
+
+  &__stars {
+    display: flex;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+    font-weight: 400;
+    color: #060b35;
+  }
+}
+
+.star {
+  background-image: url(../images/star.svg);
+  background-position: 50%;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  gap: 4px;
+  margin-right: 4px;
+
+  &:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+
+  &__word {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    color: #616070;
+  }
+
+  &__numeric {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    color: #060b35;
+  }
+}
+
+.bottom {
+  display: block;
+  text-align: center;
+  padding: 12px;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  text-decoration: none;
+  border-radius: 5px;
+  font-weight: 700;
+  background-color: #00acdc;
```

Comment:
The background color `#00acdc` is used multiple times. It would be beneficial to define this as an SCSS variable to ensure consistency and ease of maintenance.